packages/block-editor/src/store/selectors.js

```diff
@@ -2972,3 +2973,33 @@ export const isGroupable = createRegistrySelector(
 			);
 		}
 );
+
+export const getSectionsContainerClientId = createRegistrySelector(
+	( select ) =>
+		( state, postType = 'page' ) => {
```

Comment:
This is not a final API because block editor store cannot be aware of post types.

Reply:
Can we just not care? We can disable the mode for post types we don't support.

Reply:
I think we should do the opposite - just always target main even if we are in a post/page

Reply:
I think we can just see if there is a post content block. If there is then we use it, if not then we use main.

Reply:
Actually no that won't work, since you might be editing the template for a page.